### ISSUE TITLE
Plugins Translation into French

### ISSUE THREAD

**Issue by FayzB** [pannal/Sub-Zero.bundle#486]
Possible to translate the Plugins in French
I can take care of it
I only need the file to translate

**Comment by pannal** [comment]
Yep, this'll come once Plex Inc fixes their web client to send the `X-Plex-Language` header. Then we can properly translate the app.

#458

**Comment by FayzB** [comment]
Currently I started the translation by modifying the .json manually to have it on my personal PMS
GitHub link
https://github.com/FayzB/Traduction-Plugins-PMS

**Comment by pannal** [comment]
Well that's great. Keep doing that and we'll merge your progress once Plex Inc has fixed the issue.

**Comment by FayzB** [comment]
OK thanks a lot with your plugins which helps us a lot

**Comment by morpheus133** [comment]
Hi @FayzB!
Your methode is not the best for translation.
You only need to add Contents/Strings/fr.json

Check:
https://github.com/pannal/Sub-Zero.bundle/issues/458

And also need  to add all the translatable strings between  L() method.

**Comment by pannal** [comment]
@morpheus133 that's being worked on by @ukdtom. We're still deciding on a viable translation method.

**Comment by pannal** [comment]
We will be using POEditor.com for SZ's translation. Join here: http://v.ht/szi18n

**Comment by FayzB** [comment]
@morpheus133 
yes not wrong
But it allows to have the plugins in French while Plex Inc corrects the bug.
After I would correct the file

**Comment by FayzB** [comment]
Hi @pannal 
translation JPG
https://github.com/FayzB/Traduction-Plugins-PMS/tree/master/Sub-Zero.bundle/Contents/Resources

**Comment by pannal** [comment]
Oof, I'm not sure I want to include separate images for all languages to be honest. That'd increase the package size by at least 7 MB for 10 languages (and we're at 13).

**Comment by pannal** [comment]
@FayzB any news on the French translation? I've seen you join on POEditor
